### ISSUE TITLE
[Magic CLI]: nothing happens when i run any magic command

### ISSUE THREAD

**Issue by treeshateorcs** [modularml/mojo#3707]
### Issue description

i've tried `magic init`, `magic self-update`, and `magic install`. when i run any of them it's just stuck, with no output. i wait for a minute or so, then ^c

here's `strace magic init -vvvv` https://0x0.st/XUr-.txt

also, it consumes 100% of one of my cpu threads

### Steps to reproduce

- run any `magic` command

### Version Info

```shell
- magic 0.3.1 - (based on pixi 0.29.0)
- arch linux btw
- x86_64
- i'm using zsh shell, but i've tried this in bash with the same results
```

**Comment by morne** [comment]
I've had the same issue, and the same environment as yours (arch linux, x86_64, zsh).

It seems to be network related somehow. I have an openvpn client that I run to connect to a remote server. When the client is running, I get the behaviour that your are describing above. When I stop the client, then magic works correctly again. 

I've tried disabling the telemetry and enabling it, the issue persists. The only common denominator that I could find was the stopping and starting of the openvpn client.

**Comment by ForteDexe** [comment]
Same issue on Ubuntu, but it wont consume 100% of my CPU.
I use tailscale, turn it off wont help :(.

**Comment by jsoques** [comment]
Same issue here all command just freeze except for --help and --version

**Comment by towpi13** [comment]
I have this problem since months ago so I uninstalled it and move on. I'm surprised it's still not has been fixed

**Comment by ehsanmok** [comment]
Could try `rm -rf ~/.modular` and install magic again?

**Comment by ltjkoomen** [comment]
> Same issue on Ubuntu. I use tailscale, did turn it off, but no help. magic -V or -h worked though `magic 0.4.0 - (based on pixi 0.33.0)`
> 
> This version worked: `magic 0.3.0` Download from here: https://dl.modular.com/public/magic/raw
> 
> 0.3.1 didn't work, maybe related to this:
> 
> ```
> v0.3.1 (2024-10-03)
> Fixes a certification error when fetching some packages
> Fixes for telemetry data
> Added controls to disable telemetry for magic
> Print the pixi version with the magic --version command
> ```
> 
> could be because this channel is ded: `channels = ["conda-forge", "https://conda.modular.com/max"]`

I have the same findings on Ubuntu 22.04.5, using NordVPN. Magic version `0.3.0` is the last version that works while I'm connected to the VPN, starting with `0.3.1` magic CLI hangs when doing anything other than `--version` or `--help` while connected to the VPN.

**Comment by treeshateorcs** [comment]
the problem is still present with magic 0.5.1

**Comment by tristanbiesecker** [comment]
This might be the same as the issue that I opened [here](https://github.com/modular/max/issues/267), but it seems to be resolved for me in `0.6.*`.
